activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -111,6 +111,10 @@ def replica?
         @config[:replica] || false
       end
 
+      def supports_metadata_table?
+        @config.fetch(:supports_metadata_table, true)
```

Comment:
Maybe we can rename this config key to `use_metadata_table` or just `metadata_table`?

Reply:
Renamed it to `metadata_table`

Reply:
I should have chimed in yesterday when I saw this but `use_metadata_table` is better IMO. `metadata_table` reads like "does the meta table exist?" rather than "should rails create/use a metadata table".